### DISCUSSION TITLE
指纹识别的问题

### DISCUSSION THREAD

**Discussion by suxiwu1987** [hktalent/scan4all#107] [General]
在指纹识别模块下，fgConst.go 目录141 行
for _, x := range FGDataMap {
		if bD, ok := x["delete"]; ok && false == bD.(bool) {
			aN = append(aN, x)
		}
	}
这块一直都是空，根本走不进去，导致后面的指纹识别根本无法走通

**Comment by hktalent** [comment]
now
is fixed
